validator/rpc/handlers_keymanager.go

```diff
@@ -158,7 +158,7 @@ func (s *Server) ImportKeystores(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	// If any of the keys imported had a slashing protection history before, we
+	// If any of the keys importedKeymanagerKind had a slashing protection history before, we
```

Comment:
Same here, looks like search and replace accident.

There are several instances of this change in this file.